torchbenchmark/util/framework/detectron2/model_factory.py

```diff
@@ -163,22 +161,22 @@ def enable_fp16(self):
         self.example_inputs = prefetch(self.example_inputs, self.device, self.dargs.precision)
 
     def train(self):
+        batch_id = 0
```

Comment:
it feels a little suboptimal to have lost the ability to realize that the benchmark runner only knows how to run with a single input lol

Reply:
ok